packages/hint-apple-touch-icons/src/hint.ts

```diff
@@ -217,11 +240,18 @@ export default class AppleTouchIconsHint implements IHint {
         };
 
         const validate = async ({ resource }: TraverseEnd) => {
-            const pageDOM: HTMLDocument = context.pageDOM as HTMLDocument;
-            const appleTouchIcons: HTMLElement[] = getAppleTouchIcons(pageDOM.querySelectorAll('link'));
+            const pageDOM = context.pageDOM as HTMLDocument;
+            const appleTouchIcons = getAppleTouchIcons(pageDOM.querySelectorAll('link'));
+
+            const linksToManifest = pageDOM.querySelectorAll('link[rel="manifest"]').length > 0;
```

Comment:
Not blocking this PR, but since we know we're going to use this across a few hints, what do you think about making an `isPWA(dom)` helper method to share between them? In theory this would also make changing our "detection" logic easier in the future if needed.

Reply:
Issue created!